apps/web/src/services/exceptions/__tests__/index.test.ts

```diff
@@ -80,15 +86,17 @@ describe('CodedException', () => {
       expect(console.error).toHaveBeenCalledWith(err)
     })
 
-    it('logs only the error message on prod', () => {
+    // I can't figure out a way to override the IS_PRODUCTION constant
+    it.skip('logs only the error message on prod', () => {
       jest.spyOn(constants, 'IS_PRODUCTION', 'get').mockImplementation(() => true)
       logError(Errors._100)
       expect(console.error).toHaveBeenCalledWith('Code 100: Invalid input in the address field')
     })
   })
 
   describe('Tracking', () => {
-    it('tracks using Sentry on production', () => {
+    // I can't figure out a way to override the IS_PRODUCTION constant
+    it.skip('tracks using Sentry on production', () => {
```

Comment:
@usame-algan - here are the 2 tests we talked about. I couldn't make them work. If you have any ideas please let me know!

Reply:
Pushed a fix. It was some issue with how we override the process.env and the import order.